application/modules/clients/Enums/ClientTitleEnum.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+enum ClientTitleEnum: string
```

Comment:
@nielsdrost7 the solution provided is cool, but an `enum` will break backward compatibility. Enums were introduced in PHP 8.1 (https://www.php.net/manual/en/language.types.enumerations.php). Can we find an alternative solution?